tests/main/apt-hooks/task.yaml

```diff
@@ -19,7 +21,7 @@ prepare: |
     systemctl restart snapd.socket
 
 restore: |
-    mv /etc/systemd/system/snapd.service.d/local.conf.bak /etc/systemd/system/snapd.service.d/local.conf
+    rm "$CONF_OVERRIDE"
```

Comment:
I think we can use the branch as an opportunity to use `tests.cleanup defer` for the tests that are changed